src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/X509Certificates/RSAPkcs1X509SignatureGenerator.cs

```diff
@@ -39,7 +40,7 @@ internal static PublicKey BuildPublicKey(RSA rsa)
                 //
                 // This is due to one version of the ASN.1 not including OPTIONAL, and that was
                 // the version that got predominately implemented for RSA. Now it's convention.
-                new AsnEncodedData(oid, stackalloc byte[] { 0x05, 0x00 }),
+                new AsnEncodedData(oid, asnNull),
```

Comment:
This one seems like it increases allocations, since now it's creating an array instead of a stack-span, but still making the copy in the ctor.  It would reduce a copy if you passed `, skipCopy: true)` here though...

Reply:
It's not creating an array. It's using the "Implicit conversions from arrays to spans are magically created from the .TEXT segment"

https://sharplab.io/#v2:CYLg1APgAgTAjAWAFBQMwAJboMLoN7LpGYZQAs6AggE7UCGAngEICuALgHID2bASgKZ0ANkIYAKAJT5CxWQLrAA8gDtRAZQAOdZQB4mDNvwB86fugC86ZfwDu6AEYH+AbQC6+dAAYAHp4CsADRevp7oAL4A3DKyRAAaYvwSUUiyYdHo6elomBRqbHQAxgDWlCJcBZLSKTFE8kqqDJraek4mZpYAzvnFwkLlDk5uHj7+QSOhkemy8YnJqZnVRBrUAJYAbnSGmHAAbDno8XUq6lq6+oZtEukEi+HIYUA==

Reply:
Oh, C#, you're getting so hard to understand :smile: